packages/xo-web/src/xo-app/xosan/new-xosan.js

```diff
@@ -89,6 +135,10 @@ export default class NewXosan extends Component {
           return
         }
 
+        if (pool === null) {
+          return
+        }
```

Comment:
Why wait for the catalog fetch to test this?
(if it makes too many changes, you may want to open another PR for this :) )